### ISSUE TITLE
Arcgis Provider

### ISSUE THREAD

**Issue by mrelliot69** [geocoder-php/Geocoder#1165]
Is the doc updated?
When i try to use 
$provider = new \Geocoder\Provider\ArcGISList\ArcGISList($httpClient);
says class not found, but if i use
$provider = new Geocoder\Provider\ArcGISOnline\ArcGISOnline($httpClient);
it works, so im i doing something wrong or ?

**Comment by mtmail** [comment]
The directory is called ArcGisOnline so I’d say the documentation is outdated. https://github.com/geocoder-php/Geocoder/tree/master/src/Provider

**Comment by mrelliot69** [comment]
yap thats what i also tought,
question: why when using arcgis sometimes when i type Rua do, we retturns almost every result with some letters like Asd, a /n
do i need to setup another setting to work properly?

**Comment by jbelien** [comment]
> Is the doc updated?
> When i try to use
> $provider = new \Geocoder\Provider\ArcGISList\ArcGISList($httpClient);
> says class not found, but if i use
> $provider = new Geocoder\Provider\ArcGISOnline\ArcGISOnline($httpClient);
> it works, so im i doing something wrong or ?

Indeed, [`Readme.md`](https://github.com/geocoder-php/Geocoder/blob/master/src/Provider/ArcGISOnline/Readme.md) seems outdated.
Could you create a PR to update it?

> question: why when using arcgis sometimes when i type Rua do, we retturns almost every result with some letters like Asd, a /n
> do i need to setup another setting to work properly?

We don't "mess" with the result returned by the external API, we just provide a provider around it.
That issue probably comes from the API itself. Please check.
